aws/resource_aws_organizations_organization_test.go

```diff
@@ -354,3 +354,91 @@ func testFlattenOrganizationsRootPolicyTypes(t *testing.T, index int, result []m
 		}
 	}
 }
+
+func testAccAwsOrganizationsOrganizationRecreated(t *testing.T, before, after *organizations.Organization) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		if *before.Id == *after.Id {
+			t.Fatalf("Expected change to Org ID value, but both were %v", *before.Id)
+		}
+		return nil
+	}
+}
+
+func testAccAwsOrganizationsOrganization_FeatureSetForcesNew(t *testing.T) {
```

Comment:
small nitpick here: it would be helpful to keep these new tests (ForcesNew, Update) close to the existing `FeatureSet` test above